File: UsingStetho/app/src/main/java/github/nisrulz/usingstetho/MyApplication.java
package github.nisrulz.usingstetho;

import android.app.Application;

import com.facebook.stetho.Stetho;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        // http://facebook.github.io/stetho/
        Stetho.initializeWithDefaults(this);
    }
}

File: UsingStetho/app/src/main/java/github/nisrulz/example/usingstetho/MyApplication.kt
package github.nisrulz.example.usingstetho

import android.app.Application
import com.facebook.stetho.Stetho

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        // https://github.com/facebook/stetho
        // Just start your app and point your laptop browser
        // to chrome://inspect. Click the "Inspect" button to begin.
        Stetho.initializeWithDefaults(this)
    }
}